src/core/Slider/Slider.tsx

```diff
@@ -272,6 +313,7 @@ export const Slider = React.forwardRef<HTMLDivElement, SliderProps>(
       (
         event: PointerEvent,
         callback?: (values: ReadonlyArray<number>) => void,
+        skipValueChangeCheck?: boolean,
```

Comment:
Where will this be false?

Reply:
renamed `skipValueChangeCheck` to `forceCallbackToRun`